src/structures/round.ts

```diff
@@ -20,7 +20,7 @@ export default abstract class Round {
     public readonly artistAliases: string[];
 
     /** Timestamp of the creation of the Round in epoch milliseconds */
-    public readonly startedAt: number;
+    public startedAt: number;
```

Comment:
If we need seperate variable to track when audio starts playing, should create it.